doc/database/schemata/03er_table_descriptions.md

```diff
@@ -69,22 +71,26 @@ Store relations between plants.
-| **id**              | 1             |
-| **owner_id**        | 1             |
-| **name**            | My Map        | only alphanumerical values                                                                     |
-| **is_inactive**     | false         |
-| **last_visit**      | 2023-04-04    |
-| **honors**          | 0             | 0 to infinity                                                                                  |
-| **visits**          | 0             | 0 to infinity                                                                                  |
-| **harvested**       | 0             | 0 to infinity, amount of plants harvested on this map                                          |
-| **version_date**    | 2023-04-04    | the date the snapshot for this version was taken                                               |
-| **creation_date**   | 2023-04-04    |
-| **deletion_date**   | 2023-04-04    |
-| **inactivity_date** | 2023-04-04    |
-| **zoom_factor**     | 100           | value used in formula "X by X cm", e.g. 100 would mean "100 x 100 cm", range from 10 to 100000 |
-| **geo_data**        | NULL          | PostGis Geodata, location of the map                                                           |
+| **_Column name_** | **_Example_** | **_Description_**                                                                              |
+| :---------------- | :------------ | :--------------------------------------------------------------------------------------------- |
+| **id**            | 1             |
+| **owner_id**      | 1             |
+| **name**          | My Map        | only alphanumerical values                                                                     |
+| **is_inactive**   | false         |
+| **last_visit**    | 2023-04-04    |
+| **honors**        | 0             | 0 to infinity                                                                                  |
+| **visits**        | 0             | 0 to infinity                                                                                  |
+| **harvested**     | 0             | 0 to infinity, amount of plants harvested on this map                                          |
+| **privacy**       | protected     | privacy_option enum type                                                                       |
+| **description**   | Our first map |                                                                                                |
+| **creation_date** | 2023-04-04    |
+| **deletion_date** | 2023-04-04    |
+| **zoom_factor**   | 100           | value used in formula "X by X cm", e.g. 100 would mean "100 x 100 cm", range from 10 to 100000 |
+| **location**      | NULL          | PostGis Geodata, location of the map                                                           |
+| **geometry**      | FILLME        | Layout of the map.                                                                             |
```

Comment:
Not up to date with the database schema

Reply:
I think we shouldn't fully replicate everything in docu but rather only have the essence. Maybe we simply remove/reduce this?